controller/work_item_labels.go

```diff
@@ -0,0 +1,74 @@
+package controller
+
+import (
+	"github.com/fabric8-services/fabric8-wit/app"
+	"github.com/fabric8-services/fabric8-wit/application"
+	"github.com/fabric8-services/fabric8-wit/jsonapi"
+	"github.com/fabric8-services/fabric8-wit/label"
+	"github.com/fabric8-services/fabric8-wit/log"
+	"github.com/fabric8-services/fabric8-wit/workitem"
+	"github.com/goadesign/goa"
+	uuid "github.com/satori/go.uuid"
+)
+
+// WorkItemLabelsController implements the work_item_labels resource.
+type WorkItemLabelsController struct {
+	*goa.Controller
+	db     application.DB
+	config WorkItemLabelsControllerConfiguration
+}
+
+//WorkItemLabelsControllerConfiguration configuration for the WorkItemLabelsController
+type WorkItemLabelsControllerConfiguration interface {
+	GetCacheControlLabels() string
+}
+
+// NewWorkItemLabelsController creates a work_item_labels controller.
+func NewWorkItemLabelsController(service *goa.Service, db application.DB, config WorkItemLabelsControllerConfiguration) *WorkItemLabelsController {
+	return &WorkItemLabelsController{
+		Controller: service.NewController("WorkItemLabelsController"),
+		db:         db,
+		config:     config,
+	}
+}
+
+// List runs the list action.
+func (c *WorkItemLabelsController) List(ctx *app.ListWorkItemLabelsContext) error {
+	return application.Transactional(c.db, func(appl application.Application) error {
+		wi, err := appl.WorkItems().LoadByID(ctx, ctx.WiID)
+		if err != nil {
+			return jsonapi.JSONErrorResponse(ctx, goa.ErrNotFound(err.Error()))
+		}
+		var ls []label.Label
```

Comment:
same here, you can initialize an array with the length of `labelIDs`

Reply:
yes! done in https://github.com/fabric8-services/fabric8-wit/pull/1627/commits/20a09410a4cb30b338229bbad7a31ddf4431f377